src/main/java/org/fundacionjala/core/constans/ConstantsCore.java

```diff
@@ -0,0 +1,26 @@
+package org.fundacionjala.core.constans;
+
+/**
+ * [SL] This class offers constant for the core.
+ */
+public final class ConstantsCore {
```

Comment:
What if this class should be an enum instead?

Reply:
I think the best solution is change name maybe to BrowserConstans.